frontend/student.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>XBlock Accordion Student View</title>
+    <style>
+        #root {
+            width: 80%;
+            margin: auto;
+        }
```

Comment:
This is simply to give it a centered UI to make it closer to the studio modal or student view.